test/34.js

```diff
@@ -0,0 +1,20 @@
+const expect = require('chai').expect;
+//let solution = require('../solutions/34').solution;
+solution = require('../yourSolution').solution;
+
+describe('repeating elements', () => {
+  it.only('1 should return repeating elements of an array', () => {
```

Comment:
remove only. `it.('should return 1 for 1 element')`, `it.('should return 2 for 2 repeating elements')`, `it.('should return 3 for 3 repeating elements')`